roles/base_os/tasks/main.yaml

```diff
@@ -1,26 +1,26 @@
 ---
 # basic role, configures irbrc, vimrc
-
 - name: Ensure irbrc is installed for user root
   copy: >
     src=irbrc
     dest=/root/.irbrc
+  async: 0
```

Comment:
Are you trying to do a 'fire and forget' here? If so, it should be 'poll: 0'  and put a value for async. I'm looking at the docs and they don't say if 'async: 0' is a special case or not.